src/components/interface/AudioPlay.jsx

```diff
@@ -1,26 +1,82 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
+
 import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
```

Comment:
group this with the material import on line 3.

Line 1,2 together. New Line. Line 3 and 5 together. New Line, then lines 6-8 together.